### DISCUSSION TITLE
Frontend & Game Developer for Colonist.io

### DISCUSSION THREAD

**Discussion by demiculus** [pixijs/pixijs#7480] [Jobs]
- Remote
- Full-time or internship
- Flexible work hours

Hello, 

We're building https://colonist.io and alternative to Settlers of Catan. We're currently scaling our team and are looking for a Frontend/Game developer. We use TS, EJS, SCSS, PIXI. Though we are contemplating switching to React for the lobby, room, leaderboards etc.. 

If you're interested please apply through this form: https://airtable.com/shrqSxfnbzLVFnSue

You can ask any questions here or through jobs@colonist.io

**Comment by ivanpopelyshev** [comment]
> Though we are contemplating switching to React for the lobby, room, leaderboards etc..

Usually its a good idea, if you dont have UI coder who can make custom UI components :)
